Makefile

```diff
@@ -19,15 +19,36 @@ create-instance: ## Creates RDS Instance - mandatory: INSTANCE_NAME=[name];
 	echo -e "\nPassword Secret Endpoint:"
 	echo $(INSTANCE_NAME)-nonprod-dos_db_password
 
-download-sql-dump:
+download-sql-dump: ## Downloads the latest DoS database dump and gunzips it
 	# TODO: Downlaod the latest DoS database SQL dump file
 	# INPUT:
 	#		- link to an SQL dump file
+	make aws-s3-download URI=nhsd-texasplatform-service-dos-lk8s-nonprod/\
+	$(shell aws s3 ls s3://nhsd-texasplatform-service-dos-lk8s-nonprod | \
+	sort | grep "dos-pg-dump-.*-clean-PU.sql.gz" | tail -n 1 | \
+	awk '{print $$4}') \
+	FILE=/project/build/docker/data/assets/sql/dos-dump.sql.gz
+	gzip -d build/docker/data/assets/sql/dos-dump.sql
```

Comment:
Can we convert `build/docker/data/assets/sql/dos-dump.sql` to `$(DOCKER_DIR)/data/assets/sql/dos-dump.sql`?